style.css

```diff
@@ -1 +1,134 @@
-/*  Add your styling here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header-content {
+  display: grid;
+  grid-template-columns: 45% 55%;
+  background-color: rgb(12, 228, 113);
+  padding: 20px;
+  border-bottom: 1px solid #eee;
+}
+.logo-img {
+  grid-row: 1 / -1;
+  margin-left: 10px;
+}
+.logo-img img {
+  display: grid;
+  opacity: 100%;
+  width: 250px;
+  height: 200px;
+}
+.items {
+  margin: 3rem 0;
+  flex-shrink: inherit;
+  margin: 0;
+}
+.links {
+  text-transform: uppercase;
+  font-weight: 300;
+  background-color: #0d3b19;
+  padding: 5px 30px;
+  color: #eee;
+}
+.header-text {
+  grid-column: 2 / -1;
+  font-size: 30px;
+  font-weight: 300;
+  font-style: italic;
+  color: #0d3b19;
+  direction: rtl;
```

Comment:
The `direction` property is for styling languages that read right-to-left, like arabic.
To right-align english, use `text-align: right`